templates/dashboard.yaml

```
@@ -6,6 +6,9 @@ metadata:
   namespace: kubeaddons
   labels:
     kubeaddons.mesosphere.io/name: dashboard
+  annotations:
+    kubeaddons.mesosphere.io/appVersion: "1.10.1"
+    kubeaddons.mesosphere.io/endpoint: "/ops/portal/kubernetes/"
```

same as the bottom request, app name on the annotation
```suggestion
    kubeaddons.mesosphere.io/endpoint/dashboard: "/ops/portal/kubernetes/"
```